common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -600,13 +600,14 @@ def handle_problem_html_error(self, err):
 
         return html
 
-    def get_problem_html(self, encapsulate=True):
+    def get_problem_html(self, encapsulate=True, hint_index=None):
         """
         Return html for the problem.
-
-        Adds check, reset, save buttons as necessary based on the problem config and state.
+        Displays check, reset, save, and hint buttons as necessary.
+        encapsulate:         True -- embed the html in a problem <div>
```

Comment:
Hmm, not sure what happened there. And why is Displays a better word than Adds? I'll rewrite it.